3rdparty/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ add_subdirectory(libcxxrt)
 add_subdirectory(libunwind)
 add_subdirectory(musl)
 add_subdirectory(mbedtls)
+add_subdirectory(openssl)
```

Comment:
As with experimental features, can we add a specific experimental build flag for this?

Reply:
Sure. I'll mark it as experimental after bors try passes.